openproblems/tasks/chromatin_potential/methods/beta.py

```diff
@@ -49,7 +50,9 @@ def _get_annotation(adata, retries=3):
     genes = []
     for i in adata.var.index.map(lambda x: x.split(".")[0]):
         try:
-            gene = data.gene_by_id(i)
+            with warnings.catch_warnings():
+                warnings.simplefilter("ignore")
```

Comment:
This is too broad. Can you use `warnings.filterwarnings` and specify type and message="No results found for query"?

Reply:
I'm also not certain that these messages are being printed by `warnings`.